minecraft_servers/opinsel/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+    "server_name": "opinsel",
+    "nice_name": "OpInsel",
+    "direct_ip": "mc.opinsel.de",
+    "server_wildcards": [
+      "%.opinsel.de"

```

Comment:
Do you really have a wildcard?
E.g. no DNS record for "abc.opinsel.de" found

Reply:
Can be removed